api/services/handle_event_service.py

```diff
@@ -56,7 +56,7 @@ def handle_event_service(self, form_data):
                 The telco code for answered calls is 16.
                 """
                 if form_data["TelcoCode"] == str(app.config["TELCO_CODE_ANSWERED"]):
-                    data["pick_time"] = form_data["PickTime"]
+                    data["pick_time"] = form_data.get("PickTime")

```

Comment:
Used .get() so, data["pick_time"] will get the value of the "PickTime" available in the form. If there was no PickTime then the default null value will be added.